packages/manager/modules/pci/src/projects/project/private-registry/list/index.js

```diff
@@ -0,0 +1,59 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import listComponent from './list.component';
+
+const moduleName = 'ovhManagerPciProjectPrivateRegistryList';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+  ])
+  .config(/* @ngInject */($stateProvider) => {
+    $stateProvider.state('pci.projects.project.private-registry.list', {
+      url: '/list',
```

Comment:
You should have directly `/` not `/list`

Reply:
No error if you `import { DELETE_CONFIRMATION_INPUT } from './private-registry.constants.js'; `
It will throw an error if you `import constants from './private-registry.constants.js';`